src/App.js

```diff
@@ -1,14 +1,78 @@
-import './App.css';
-import Education from './components/Education';
-import Experience from './components/Experience';
-import General from './components/General';
+import { useState } from 'react';
+import styles from './app.module.css';
```

Comment:
Separating lib imports from local imports helps to read code easier. I personally separate as:
1. NPM libraries
2. Your own libraries from some kind of artifacts (you don't have those, so skip)
3. Utilities imports (you also don't have it, but it might be something like `import dayjs from '../../core/dayjs';` and similar'
4. Local imports

All of these is separated by whitespace